javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -421,13 +470,46 @@ class Driver extends webdriver.WebDriver {
    */
   setFileDetector() {
   }
+
+  /**
+   * Get the context that is currently in effect.
+   *
+   * @return {Context} Current context.
+   */
+  get context() {
```

Comment:
In the constructor, should we store whether marionette is being used and fail fast here if not?